tests/tools/oesign/test-inputs/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# Copy all manually authored test configs to output folder
+add_custom_command(
+  OUTPUT valid.conf
+  COMMAND cmake -E copy ${CMAKE_CURRENT_SOURCE_DIR}/valid.conf
+          ${CMAKE_CURRENT_BINARY_DIR})
+
+# Generate an empty config file
+add_custom_command(OUTPUT empty.conf
+                   COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/make-oesign-config.py 0)
+
+# Generate a config file with negative value
+add_custom_command(OUTPUT negative_num_heap_pages.conf
+                   COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/make-oesign-config.py 1)
+
+add_custom_target(oesign_test_configs ALL DEPENDS valid.conf empty.conf
+                                                  negative_num_heap_pages.conf)
+
+# Generate a valid SGX signing key-pair
+add_custom_command(
```

Comment:
I vaguely remember you should have a depends here. See https://github.com/openenclave/openenclave/pull/2226/files

Reply:
@radhikaj Thanks for the review, Radhika! There is already a custom target that requires the build of the files generated by this custom command, see lines 34-36. They are being consolidated into a single target to make it easier to reference (i.e. downstream projects only need to specify a single target dependency)